src/com/pentalog/pentastagiu/homework2/problema2/Basket.java

```diff
@@ -0,0 +1,41 @@
+package com.pentalog.pentastagiu.homework2.problema2;
+
+public class Basket {
+    private Customer customer;
```

Comment:
eu as vedea putin diferita relatia Basket-Customer: e mai logic sa spunem ca "un customer are un cos de cumparaturi" decat "un cos de cumparaturi are un customer", nu? de aceea as adauga `private Basket basket = new Basket();` si poti adauga in customer logica legata de membership: iei pretul total al cosului si in functie de tipul de membership faci o recalculare, aplici reducerea